include/ogdf/basic/GraphList.h

```diff
@@ -159,6 +159,24 @@ class OGDF_EXPORT GraphListBase {
 		(m_tail = pPred)->m_next = nullptr;
 	}
```

Comment:
This method could just use the new iterator-based one (assuming that all our lists have `begin` and `end`), right? That would prevent some code duplication.